config/modelConfig_test.go

```diff
@@ -0,0 +1,50 @@
+package config
```

Comment:
This seems to be duplicating the properties package, can we reuse the logic in there, as that is also fully tested on it's YAML parsing ability

Reply:
Would we need to make properties support model group on config/`dbt_project.yml`?

Reply:
I think that would be a worth while change to add that support, I'd love to merge the config yaml parsing into the properties one. What do you think?